test/Query.js

```diff
@@ -1166,13 +1166,13 @@ describe("Query", () => {
 		it("Should send correct request on query.exec", async () => {
```

Comment:
For these query tests. It looks like we added a bunch more tests to Scan that didn't get added to Query. Can we make sure they are copied over here as well?